1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,151 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @2903a
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        # Save button is removed as Project API demands a PM
+        And they see a Next Button
+
+    @2905
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard" 
+        # but enables a Curator to edit projects where he's not an participant
+        Given User opens Landing Page for Project
+        And User is Curator on Project's Project Owner's Institution
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+
+    @2906a
+    Scenario Outline: User opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has one of these role in the project:
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Button Add Project Participant
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Button "<Button>" decided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Status    | Button                   |
+            | Draft     | Save draft               |
+            | Published | Update published project |
```

Comment:
```suggestion
            | Role | Status    | Button                   |
            | Curator               | Draft     | Save draft               |
            | Curator               | Published | Update published project |
            | Project Owner         | Draft     | Save draft               |
            | Project Owner         | Published | Update published project |
            | Project Manager       | Draft     | Save draft               |
            | Project Manager       | Published | Update published project |
            | Local Project Manager | Draft     | Save draft               |
            | Local Project Manager | Published | Update published project |
            
```

Reply:
denne blir veldig krøkkete da, siden Role ikke påvirker Status/Button på noe vis. Det var her vi ville separere de to for å forenkle scenarioet :/
Knappen skal være "Save draft" hvis draft, og "Update published project" hvis publisert. Alle rollene skal se det samme

Reply:
Rollene bestemmer tilgang til å endre på prosjekt.
Prosjektet sin status bestemmer navn på lagre-knapp.
Den eine er ikkje avhengig av den andre.